fog/overseer/server/src/config.rs

```diff
@@ -0,0 +1,58 @@
+// Copyright (c) 2018-2022 The MobileCoin Foundation
+
+use mc_common::ResponderId;
+use mc_fog_sql_recovery_db::SqlRecoveryDbConnectionConfig;
+use mc_fog_uri::FogIngestUri;
+use serde::Serialize;
+use structopt::StructOpt;
+
+/// StructOpt configuration options for an Overseer Server
+#[derive(Clone, Serialize, StructOpt)]
+pub struct OverseerConfig {
+    /// Host to listen on.
+    #[structopt(long, default_value = "127.0.0.1")]
+    pub listen_host: String,
+
+    /// Port to start webserver on.
+    #[structopt(long, default_value = "9090")]
+    pub listen_port: u16,
+
+    /// Local Ingest Node ID
+    #[structopt(long)]
+    pub local_overseer_node_id: ResponderId,
```

Comment:
Ah, I didn't realize that ResponderId corresponded to an enclave. I still want to have some sort of ID that corresponds to the Fog Overseer instance (one day, we'll be able to run multiple overseer instances at once), so perhaps a simple String would suffice instead of a ResponderId?

Reply:
Is the hostname of the machine not good enough?

Reply:
I'd also suggest that there's no need to future-proof this at the moment. Multiple instances will be a big enough change that this can be introduced at that point.

Reply:
SG, just removed the usage of ResponderId.